src/Agent.Sdk/Knob/AgentKnobs.cs

```diff
@@ -716,5 +716,14 @@ public class AgentKnobs
             "Checks if the PSModulePath environment variable contains locations specific to PowerShell Core.",
             new EnvironmentKnobSource("AZP_AGENT_CHECK_PSMODULES_LOCATIONS"),
             new BuiltInDefaultKnobSource("false"));
+
+        public static readonly Knob UseDockerStdinPasswordOnWindows = new Knob(
+            nameof(CheckPsModulesLocations),
+            "If true, use --password-stdin for docker login on Windows.",
+            new EnvironmentKnobSource("AZP_AGENT_USE_DOCKER_STDIN_PASSWORD_WINDOWS"),
+            new RuntimeKnobSource("AZP_AGENT_USE_DOCKER_STDIN_PASSWORD_WINDOWS"),
+            new EnvironmentKnobSource("AZP_AGENT_USE_DOCKER_STDIN_PASSWORD_WINDOWS"),
+            new PipelineFeatureSource("UseDockerStdinPasswordOnWindows"),
+            new BuiltInDefaultKnobSource("false"));
```

Comment:
```suggestion
            new EnvironmentKnobSource("AZP_AGENT_USE_DOCKER_STDIN_PASSWORD_WINDOWS"),
            new RuntimeKnobSource("AZP_AGENT_USE_DOCKER_STDIN_PASSWORD_WINDOWS"),
            new PipelineFeatureSource("UseDockerStdinPasswordOnWindows"),
            new BuiltInDefaultKnobSource("false"));
```

P.S. If this is FF-only change, are we actually need to have system env variable source?

Reply:
Or, if this something which user should decide by himself, then I believe we don't need the `PipelineFeatureSource`

Reply:
Removed EnvironmentKnobSource, thanks